static/js/settings_config.js

```diff
@@ -110,15 +110,20 @@ export const create_stream_policy_values = {
         description: $t({defaultMessage: "Admins"}),
     },
     by_full_members: {
-        order: 2,
+        order: 3,
         code: 3,
         description: $t({defaultMessage: "Admins and full members"}),
     },
     by_members: {
-        order: 3,
+        order: 4,
         code: 1,
         description: $t({defaultMessage: "Admins and members"}),
     },
+    by_moderators_only: {
+        order: 2,
+        code: 4,
+        description: $t({defaultMessage: "Admins and moderators"}),
+    },
```

Comment:
Can you add the new one just after `by_admins_only`?  Seems like it's nice to have the order be the same.